libtelemetry/src/main/java/com/mapbox/android/telemetry/CertificateBlacklist.java

```diff
@@ -240,4 +247,26 @@ private String determineConfigEndpoint() {
     url = url.replace(HTTPS, EMPTY_STRING);
     return url.split(BACKSLASH);
   }
+
+  private void setupMetrics(long serverTime, String responseString) {
+    MetricUtils.setConfigResponse(responseString);
+    MetricUtils.calculateTimeDiff(serverTime);
+  }
+
+  private long obtainServerTime(Response response) {
+    Headers headers = response.headers();
+
+    String dateString = headers.get("Date");
+
+    SimpleDateFormat dateFormat = new SimpleDateFormat(DATE_FORMAT);
+    Date serverDate = new Date();
```

Comment:
what's the benefit of instantiating date object here? if date is invalid would you rather throw exception?